<launch>

    <arg name="map_frame" default="map"/>
    <arg name="odom_frame" default="odom"/>
    <arg name="base_frame" default="base_link"/>
    <arg name="audio_trigger_topic" default="/audio_trigger"/>
    <arg name="audio_status_topic" default="/audio_status"/>

    <!-- Waypoints -->
    <arg name="initial_pose_x" value="38.8"/>
    <arg name="initial_pose_y" value="16"/>
    <arg name="initial_pose_heading" value="0"/>
    <arg name="waypoints_file_folder" value="/home/elliottwhite/proj515_ws/src/waypoint_navigation/waypoints"/>

    <!-- Map Server --> 
    <node name="map_server" pkg="map_server" type="map_server" args="/home/elliottwhite/proj515_ws/map/303/303.pgm 0.05"/>

    <!-- Waypoint navigation -->
    <node pkg="waypoint_navigation" type="waypoint_navigation" name="waypoint_navigation" output="screen">
        <param name="map_frame" value="$(arg map_frame)"/>
        <param name="odom_frame" value="$(arg odom_frame)"/>
        <param name="base_link_frame" value="$(arg base_frame)"/>
        <param name="audio_trigger_topic" value="$(arg audio_trigger_topic)"/>
        <param name="audio_status_topic" value="$(arg audio_status_topic)"/>
        <param name="initial_pose_x" value="$(arg initial_pose_x)"/>
        <param name="initial_pose_y" value="$(arg initial_pose_y)"/>
        <param name="initial_pose_heading" value="$(arg initial_pose_heading)"/>
        <param name="waypoints_file_folder" value="$(arg waypoints_file_folder)"/>
    </node>


</launch>